### DISCUSSION TITLE
Renaming projects in QField

### DISCUSSION THREAD

**Discussion by Skipper-is** [opengisch/QField#5211] [Ideas]
This would be very handy, being able to rename projects within QField. I have a number of project templates I give to surveyors when they go out in the field, but after a little while they tend to end up with 3 projects all with the same name (badgers_1, badgers_2 etc), would be handy if we could rename them in the app.

**Comment by ZsanettMed** [comment]
Hello @Skipper-is, thanks for sharing your idea. I've added it to our [QField Idea Platform](https://ideas.qfield.org/app-feature-requests/p/renaming-projects-in-qfield) so you and other users can vote on it. This helps us set priorities for our development roadmap. 

PS. I'm closing this discussion here since we are moving all requests to the QField Idea Platform. :)

Best, 
Zsanett
